packages/cli-repl/src/mongosh-repl.ts

```diff
@@ -137,11 +137,20 @@ class MongoshNodeRepl implements EvaluationListener {
         this.insideAutoComplete = true;
         try {
           // Merge the results from the repl completer and the mongosh completer.
-          const [ [replResults], [mongoshResults] ] = await Promise.all([
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
+          const [ [replResults], [mongoshResults, _, mongoshResultsExclusive] ] = await Promise.all([
```

Comment:
Shouldn't `[mongoshResults,, mongoshResultsExclusive]` work too?

Reply:
@mcasimir Yes, I just didn’t think of that :)